src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
 public class MainApp {
+    private final Machine truck = new Truck();
```

Comment:
It's better to create an array Machine[] machines = new Machine[3];  in method printInfo() and add each type of machine there

Reply:
Changed, thanks